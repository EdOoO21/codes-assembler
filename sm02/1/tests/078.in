-804 -386 893 361 495 785 -473 522 -125 843 -931 342 -699 812 334 -180 581 358 -614 889 426 -36 -225 997
