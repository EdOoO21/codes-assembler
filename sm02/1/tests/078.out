997
-225
-36
426
889
-614
358
581
-180
334
812
-699
342
-931
843
-125
522
-473
785
495
361
893
-386
-804
